modules/ROOT/pages/flex-gateway-permissions.adoc

```diff
@@ -0,0 +1,21 @@
+= Permissions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]s
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To see and use Flex Gateway Connected Mode capabilities, organizations must have a license.
+The entitlement is provided by sales operations through an admin console.
+
+Entitled organizations see a *Flex Gateway* tab in the Runtime Manager UI, and two new types of Runtime Manager permissions:
+
+[%header,cols="20a,60a"]
+|===
+|Permission |Description
+|Manage Gateways |Access to create, update, and delete Flex Gateway resources. A user with this permission will be able to add gateways, unregister/delete gateways, and add/edit configurations.
```

Comment:
will be > is

able to add gateways, unregister/delete gateways, and add/edit configurations > 
able to add, unregister, and delete gateways, and add or edit configurations.